src/overlays/actors/ovl_En_Bigslime/z_en_bigslime.c

```diff
@@ -54,289 +243,2910 @@ const ActorInit En_Bigslime_InitVars = {
+            this->actor.bgCheckFlags |= 2;
+        }
+
+        this->actor.bgCheckFlags |= 1;
+    } else {
+        this->actor.bgCheckFlags &= ~1;
+    }
+
+    if ((this->actionFunc != EnBigslime_Freeze) || !(this->actor.bgCheckFlags & 1)) {
+        worldPosX = this->actor.world.pos.x;
+        vtxMaxX = vtxMax->x;
+        if (GBT_ROOM_5_MAX_X < (worldPosX + vtxMaxX)) {
+            this->actor.bgCheckFlags |= 8;
+            this->actor.world.pos.x = GBT_ROOM_5_MAX_X - vtxMaxX;
+        } else {
+            vtxMinX = vtxMin->x;
+            if ((worldPosX + vtxMinX) < GBT_ROOM_5_MIN_X) {
+                this->actor.world.pos.x = GBT_ROOM_5_MIN_X - vtxMinX;
+                this->actor.bgCheckFlags |= 8;
+            }
+        }
+
+        worldPosZ = this->actor.world.pos.z;
+        vtxMaxZ = vtxMax->z;
+        if (GBT_ROOM_5_MAX_Z < (worldPosZ + vtxMaxZ)) {
+            this->actor.bgCheckFlags |= 8;
+            this->actor.world.pos.z = GBT_ROOM_5_MAX_Z - vtxMaxZ;
+        } else {
+            vtxMinZ = vtxMin->z;
+            if ((worldPosZ + vtxMinZ) < GBT_ROOM_5_MIN_Z) {
+                this->actor.world.pos.z = GBT_ROOM_5_MIN_Z - vtxMinZ;
+                this->actor.bgCheckFlags |= 8;
+            }
+        }
+    }
+}
+
+void EnBigslime_UpdateBigslimeCollider(EnBigslime* this, GlobalContext* globalCtx) {
+    Vtx* dynamicVtx;
+    f32 xzDist;
+    s16 vtxRingMaxY[BIGSLIME_NUM_RING_VTX];
+    s16 vtxRingMinY[BIGSLIME_NUM_RING_VTX];
+    f32 vtxRingMaxXZDist[BIGSLIME_NUM_RING_VTX];
+    Cylinder16* dim;
+    f32 maxScaleXScaleZ = (this->actor.scale.z < this->actor.scale.x ? this->actor.scale.x : this->actor.scale.z);
+    s32 i;
+    s32 j;
+
+    for (i = 0; i < BIGSLIME_NUM_RING_VTX; i++) {
+        vtxRingMaxY[i] = -30000;
+        vtxRingMinY[i] = 30000;
+        vtxRingMaxXZDist[i] = 0.0f;
+
+        for (j = sVtxRingStartIndex[i]; j < sVtxRingStartIndex[i + 1]; j++) {
+            dynamicVtx = &sBigslimeDynamicVtx[this->dynamicVtxState][j];
+            if (vtxRingMaxY[i] < dynamicVtx->n.ob[1]) {
+                vtxRingMaxY[i] = dynamicVtx->n.ob[1];
+            }
+
+            if (vtxRingMinY[i] > dynamicVtx->n.ob[1]) {
+                vtxRingMinY[i] = dynamicVtx->n.ob[1];
+            }
+
+            xzDist = sqrtf(SQ(dynamicVtx->n.ob[0]) + SQ(dynamicVtx->n.ob[2]));
+            if (vtxRingMaxXZDist[i] < xzDist) {
+                vtxRingMaxXZDist[i] = xzDist;
+            }
+        }
+    }
+
+    for (i = 0; i < BIGSLIME_NUM_RING_FACES; i++) {
+        dim = &this->bigslimeCollider[i].dim;
+        dim->pos.y = (vtxRingMinY[i + 1] * this->actor.scale.y) + this->actor.world.pos.y;
+        dim->height = (vtxRingMaxY[i] - vtxRingMinY[i + 1]) * this->actor.scale.y;
+        dim->radius = maxScaleXScaleZ *
+                      (vtxRingMaxXZDist[i] < vtxRingMaxXZDist[i + 1] ? vtxRingMaxXZDist[i + 1] : vtxRingMaxXZDist[i]);
+        dim->height = dim->height <= 0 ? 1 : dim->height;
```

Comment:
```suggestion
        dim->height = CLAMP_MIN(dim->height, 1);
```
Any chance this matches?